### ISSUE TITLE
Omega 2 and general outstanding issues

### ISSUE THREAD

**Issue by pieman64** [OnionIoT/Onion-Console#50]
@OnionIoT and @greenbreakfast can you please advise what Omega 2 updates you still have to do to your code and then users can compare your list with the bugs that they have found.

The known issues list at https://docs.onion.io/omega2-docs/known-firmware-issues.html#known-firmware-issues seems to be missing things like one-wire bug and GPIO tool showing Omega rather than Omega 2 etc.

**Comment by sfdgsrsgrrrrrrrrrrrrr** [comment]
Yes REALY MISSING "1-Wire" interfaca, I can't use Temperature sensor DS18S20 :-((
